bin/install/upload_to_s3.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -x
+
+COSMOS_VERSION="0.4.0"
+
+cosmos_cli()
+{
+        # result is the larger of the two versions
+        # convert the str to numbers, sort, and return the larger
+        result=$(echo -e "$COSMOS_VERSION\n$TAG_VERSION" | sed '/^$/d' | sort -nr | head -1)
+        # if TAGGED_VERSION >= COSMOS_VERSION we want to use cosmos cli
+        [[ "$result" = "$TAG_VERSION" ]]
+}
+
+if cosmos_cli ; then
+        aws s3 --region=us-east-1 cp \
+        %aws.bash_script_to_upload% \
+        %aws.bash_destination_url%
+
+        aws s3 --region=us-east-1 cp \
+        %aws.powershell_script_to_upload% \
+        %aws.powershell_destination_url%
+else
+        aws s3 --region=us-east-1 cp \
+        %aws.legacy_bash_script_to_upload% \
+        %aws.legacy_bash_destination_url%
+
+        aws s3 --region=us-east-1 cp \
+        %aws.legacy_powershell_script_to_upload% \
```

Comment:
The file source locations are known let's hardcode them here to minimize all of the configuration options.